### DISCUSSION TITLE
Better visualization

### DISCUSSION THREAD

**Discussion by Dipeshscofield** [gephi/gephi#2776] [Q&A]
How to visualize 49 nodes and 1516 edges in the best possible way. Like the layout and all the properties so that it is more clearer

**Comment by mbastian** [comment]
Hi @Dipeshscofield that seems like quite a high degree per node so most likely it will not look very insightful. If your edges have a weight and it makes sense for your use-case you could try to filter those edges.

You could also checkout [this plugin](https://gephi.org/plugins/#/plugin/circularlayout) as it probably fits quite well with this number of nodes.
